docs/Project.toml

```diff
@@ -6,6 +6,6 @@ Documenter = "e30172f5-a6a5-5a46-863b-614d45cd2de4"
 Latexify = "23fbe1c1-3f47-55db-b15f-69d7ec21a316"
 ModelingToolkit = "961ee093-0014-501f-94e3-6117800e7a78"
 Plots = "91a5bcdd-55d7-5caf-9e0b-520d859cae80"
-
+Graphviz_jll = "3c863552-8265-54e4-a6dc-903eb78fde85"
```

Comment:
When you update this, please take out Graphviz as a required dependency. The jll is still too wonky (for example, saving doesn't work for svgs or pdfs on aarch64 Macs currently).

Reply:
@isaacsas  Sorry, got caught up in other work and didn't notice this untill now. What are your views on the rest of tutorial ? Do we need to add anything ?

Reply:
It looks nice.. Especially now that arguments of these functions take only `ReactionSystem` as input.
I did not understand though why the output of `complexgraph(rn)` mentioned in this [section](https://github.com/yewalenikhil65/Catalyst.jl/blob/upstreamCopy/docs/src/tutorials/reaction_network_representation.md#reaction-complex-representation) shows red arrow from `2*A + 3*B --> A + 2*C + D` ? I mean we had created such arrow only if there are multiple reactions in same substrates and products with different reaction rates, if i remember correctly.

Reply:
So black arrow should be mass action.